dashboard/config/routes.rb

```diff
@@ -79,12 +79,24 @@ module OPS
   get '/u/:id', to: redirect('/c/%{id}')
   get '/u/:id/:action_id', to: redirect('/c/%{id}/%{action_id}')
 
+  # These links should no longer be created (August 2017), though we will continue to support
+  # existing links. Instead, create /r/ links.
```

Comment:
Does this mean these will continue to be accessible even after the user is hard-deleted, once the URL is out in the world?

What does it mean they "should no longer be created"? Does that mean we won't display them anywhere? It seems these `/c/` urls will still work, even for new level_sources, if a sophisticated user decodes one of the `/r/` urls and extracts the level source id. Am I missing something?

Are we ok with that loophole, making the /r/ urls stop working as-is, but still leaving the level source accessible with a bit of url hacking?